lib/vdc/BOVCollection.cpp

```diff
@@ -103,6 +115,11 @@ int BOVCollection::Initialize(const std::vector<std::string> &paths)
 
         header.open(paths[i]);
         if (header.is_open()) {
+            if (getFileSize(paths[i]) > 1000) {
+                SetErrMsg(("BOV header file larger than 1MB.  This text file shouldn't need to be larger than a few KB." + paths[0]).c_str());
```

Comment:
I think you meant 1KB here?

Reply:
Good catch @shaomeng. Can you confirm the size here, @sgpearse, and then I'll accept.

Reply:
Fixed.  Thanks sam.